Makefile

```diff
@@ -98,6 +98,10 @@ check-docs-source-in-git: docs
 	    exit 1; \
 	fi
 
+.PHONY: doc8
+doc8:
+	doc8 --config="doc8.ini"
```

Comment:
--config is redundant b/c doc8.ini in the current directory is the default name for the config file

Reply:
The path(s) to examine are given as positional parameter to the doc8 command, so `doc8 docs/source/`